core/src/main/scala/org/apache/spark/metrics/sink/JmxSink.scala

```diff
@@ -19,7 +19,8 @@ package org.apache.spark.metrics.sink
 
 import java.util.Properties
 
-import com.codahale.metrics.{JmxReporter, MetricRegistry}
+import com.codahale.metrics.jmx.JmxReporter
+import com.codahale.metrics.MetricRegistry
```

Comment:
No other changes in this file, why do we need these two imports on this level specifically?

Reply:
Because of the dependency version changes.